build

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
It's really simple:

$ cat ./Makefile
```Makefile
PWD := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))

GOPKG = github.com/bndw/pick
GOPATH = "$(CURDIR)/vendor:$(CURDIR)"

BIN_DIR = /usr/local/bin
INSTALL = install

all: build

goget:
	GOPATH=$(GOPATH) go get github.com/rogpeppe/godeps
	GOPATH=$(GOPATH) $(CURDIR)/vendor/bin/godeps -u dependencies.tsv
	mkdir -p $(shell dirname "$(CURDIR)/vendor/src/$(GOPKG)")
	rm -f $(CURDIR)/vendor/src/$(GOPKG)
	ln -sf $(PWD) $(CURDIR)/vendor/src/$(GOPKG)

build: goget
	GOPATH=$(GOPATH) go build -o bin/pick .

test: goget
	GOPATH=$(GOPATH) go test -v ./...

install:
	@echo "Installing pick to $(BIN_DIR)/pick"
	$(INSTALL) -c bin/pick $(BIN_DIR)/pick

uninstall:
	rm -f $(BIN_DIR)/pick

clean:
	rm -rf vendor/
	rm -rf bin/

.PHONY: all goget build test install uninstall clean
```

$ cat dependencies.tsv
```
github.com/atotto/clipboard	git	bb272b845f1112e10117e3e45ce39f690c0001ad	2016-02-19T03:44:21Z
github.com/spf13/cobra	git	8e91712f174ced10270cf66615e0a9127e7c4de5	2016-01-17T03:09:38Z
github.com/spf13/pflag	git	7f60f83a2c81bc3c3c0d5297f61ddfa68da9d3b7	2015-12-18T13:47:03Z
golang.org/x/crypto	git	3760e016850398b85094c4c99e955b8c3dea5711	2016-01-13T19:11:26Z
```

- Build via `$ make` / `$ make build`
- Test via `$ make test `
- Install via `$ make install``
- Uninstall via `$ make uninstall`
- Clean via `$ make clean`